qemu/tests/cfg/edk2_basic.cfg

```diff
@@ -0,0 +1,20 @@
+- edk2_basic:
+    virt_test_type = qemu
+    only Linux
+    type = edk2_basic
```

Comment:
For edk2, only available for aarch64 and x86 q35.

Reply:
Corrected